textsubs.lic

```diff
@@ -890,4 +890,302 @@ TextSubs.add('Your skill in (.*) is at a zenith of enlightenment\.', 'Your skill
 # Commas when dealing with coins
 TextSubs.add('\d{1,3}(?=(\d{3})+(?!\d).*(Kronar|Dokora|Lirum))', '\&,')
 
+# Scroll subs to add guild name
+TextSubs.add('"Strange Arrow."', '"Strange Arrow(Analagous)."')
```

Comment:
In a regex, putting something in `()` will _capture_ it. Then, in the output, `\1` will be replaced with the value of the first captured thing. Likewise, for `(first)(second)`, `\2` will be `second`.